.dockerignore

```diff
@@ -9,3 +9,10 @@ artifacts
 clients/client-rust/target
 ui/build
 temp
+workers/generic-worker/generic-worker-*
```

Comment:
those binaries were added to gitignore because they can be accidentally copied into the docker container during build times and may conflict with binaries that are built inside the container